autoload/sy.vim

```diff
@@ -135,6 +99,43 @@ function! sy#verbose(msg, ...) abort
   endif
 endfunction
 
+" #set_autocmds {{{1
+function! sy#set_autocmds() abort
+  augroup signify
+    autocmd!
```

Comment:
I think this should be
```viml
autocmd * <buffer>
```
Tested on my machine this preserves desired per-buffer behavior.

Reply:
Yup!

Reply:
For a potential `:SignifyEnableAll`, we probably need to change this block to use `<buffer=bufnr>` instead of just `<buffer>` as well.